### PR TITLE
Fix replication frontend issues

### LINKED ISSUE #7372: Wrong error message is shown to users when creating registries
The error message and the real error don't match.
create_registry

### PR DESCRIPTION
**Pull request by pureshine**
fix #7372 #6808 
1、Fix wrong error message is shown to users when creating registries
2、Add an override field when creating a policy
3、Execution lists and details pages show total and stopped information
Signed-off-by: FangyuanCheng <fangyuanc@vmware.com>

### CODE REVIEW

File: src/portal/lib/src/create-edit-endpoint/create-edit-endpoint.component.html [hunk 1]
```diff
@@ -41,7 +41,7 @@ <h3 class="modal-title">{{modalTitle}}</h3>
           <textarea type="text" class="inputWidth" row=3 name="description" [(ngModel)]="target.description"></textarea>
         </div>
         <!-- Endpoint Url -->
-        <div class="form-group">
+        <div class="form-group" [hidden]="target.type === 'dockerHub'">
```
Comment (kofj) [inline src/portal/lib/src/create-edit-endpoint/create-edit-endpoint.component.html]: Some registry not support too, like huawei SWR.

Comment (pureshine) [reply src/portal/lib/src/create-edit-endpoint/create-edit-endpoint.component.html]: So how should I judge whether it supports it?

Comment (kofj) [reply src/portal/lib/src/create-edit-endpoint/create-edit-endpoint.component.html]: We'll discuss this tonight in the Replication Workgroup meeting up.

Comment (pureshine) [reply src/portal/lib/src/create-edit-endpoint/create-edit-endpoint.component.html]: OK ,then this pr is temporarily not making changes here.

File: src/portal/lib/src/replication/replication-tasks/replication-tasks.component.ts [hunk 2]
```diff
@@ -128,6 +139,24 @@ export class ReplicationTasksComponent implements OnInit {
         this.totalCount = res.length;
         this.tasks = res; // Keep the data
         this.taskItem = this.tasks.filter(tasks => tasks.resource_type !== "");
+        if (!this.timerDelay) {
+          this.timerDelay = timer(10000, 10000).subscribe(() => {
+            let count: number = 0;
+            this.tasks.forEach(tasks => {
+              if (
+                tasks.status === "InProgress"
```
Comment (zhoumeina) [inline src/portal/lib/src/replication/replication-tasks/replication-tasks.component.ts]: abstract to a const.

**Pull request merged**